packages/shared/src/components/Logo.tsx

```diff
@@ -19,6 +20,22 @@ const logoPositionToClassName: Record<LogoPosition, string> = {
   [LogoPosition.Relative]: classNames('relative mt-0.5', 'laptop:mt-0'),
   [LogoPosition.Initial]: '',
 };
+interface LogoSvgElemProps {
+  className?: string;
+  src?: string;
+  fallback: typeof LogoText | typeof LogoIcon;
+}
+
+const LogoSvgElem = ({
+  className,
+  src,
+  fallback: FallbackElem,
+}: LogoSvgElemProps): ReactElement => {
+  if (src) {
+    return <SVG src={src} className={className} />;
```

Comment:
Do we need a lib, we can just use `<img />` to load it up?

Reply:
Not the way we do it today. `<img` element does not allow us to change the fill or stroke color with CSS, as we do now.

Reply:
For what do we use that?

Reply:
Replacing with an `<img` element, based on latest sync with Chris and Nensi